Cache/Requests.lean

```diff
@@ -129,9 +129,24 @@ def downloadFiles (hashMap : IO.HashMap) (forceDownload : Bool) (parallel : Bool
       IO.Process.exit 1
   else IO.println "No files to download"
 
+def checkForToolchainMismatch : IO Unit := do
+  let mathlibToolchainFile := IO.mathlibDepPath / "lean-toolchain"
+  let downstreamToolchain ← IO.FS.readFile "lean-toolchain"
+  let mathlibToolchain ← IO.FS.readFile mathlibToolchainFile
+  if !(mathlibToolchain.trim = downstreamToolchain.trim) then
+    IO.println "Dependency Mathlib uses a different lean-toolchain"
+    IO.println s!"Project uses {downstreamToolchain.trim} | Mathlib uses {mathlibToolchain.trim} "
```

Comment:
```suggestion
    IO.println "Dependency Mathlib uses a different lean-toolchain"
    IO.println s!"  Project uses {downstreamToolchain.trim}"
    IO.println s!"  Mathlib uses {mathlibToolchain.trim} "
```
so that the two appear column-aligned and are easy to compare